frontend/src/main.tsx

```diff
@@ -1,12 +1,30 @@
 import ReactDOM from 'react-dom/client';
 import App from './App';
+import Post from './pages/Post';
 import './index.css';
 import { QueryClientProvider, QueryClient } from 'react-query';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import CreatePost from './pages/CreatePost';
 
 const queryClient = new QueryClient();
 
+const router = createBrowserRouter([
```

Comment:
Please add all routes for the elements located in the header bar as we are getting unhandled errors when clicking on the, for example, "About" page.